### PR TITLE
feat(web-client): add delete user account functionality

### PR DESCRIPTION
**Pull request by IscoRuta98**

### CODE REVIEW

**Comment by Claire-Bear** [comment]
Task linked: [CU-2b0107c Delete user](https://app.clickup.com/t/2b0107c)

File: web-client/src/app/services/xrpl.service.ts [hunk 1]
```diff
@@ -116,8 +116,8 @@ export class XrplService {
       async (client) => await client.getBalances(address)
     );
   }
-
-  async createUnsignedTransaction(
+  // Rename below to createUnsignedPaymentTransaction
```
Comment (longtomjr) [inline web-client/src/app/services/xrpl.service.ts:119]:
```suggestion

```
This is probably no longer needed

File: web-client/src/app/services/xrpl.service.ts [hunk 2]
```diff
@@ -133,6 +133,22 @@ export class XrplService {
...
+  ): Promise<xrpl.AccountDelete> {
+    const unpreparedTx: xrpl.AccountDelete = {
+      Account: fromAddress,
+      TransactionType: 'AccountDelete',
+      Destination: toAddress,
+    };
+    const tX = await this.withConnection(
+      async (client) => await client.autofill(unpreparedTx)
+    );
+    //tX.LastLedgerSequence = Tx.LastLedgerSequence! + 10
```
Comment (longtomjr) [inline web-client/src/app/services/xrpl.service.ts:148]:
```suggestion

```

File: web-client/src/app/views/delete-user/delete-user.page.ts [hunk 3]
```diff
@@ -0,0 +1,260 @@
...
+    {
+      title: 'Enter address manually',
+      icon: faKeyboard,
+      action: 'presentAddressModal',
+    },
+    // {
+    //   title: 'Share my wallet address',
+    //   icon: faLink,
+    //   disabled: true,
+    // },
```
Comment (longtomjr) [inline web-client/src/app/views/delete-user/delete-user.page.ts:53]:
```suggestion
```

File: web-client/src/app/views/delete-user/delete-user.page.ts [hunk 4]
```diff
@@ -0,0 +1,260 @@
...
+
+  async onRefresh(): Promise<void> {
+    await withLoadingOverlayOpts(
+      this.loadingController,
+      { message: 'Refreshing…' },
+      async () => await this.refreshWalletData()
+    );
+  }
+
+  async refreshWalletData(): Promise<void> {
```
Comment (longtomjr) [inline web-client/src/app/views/delete-user/delete-user.page.ts:143]: In the future the code duplicated between the delete-user page and the wallet page should be moved to a service or other components that can be shared. I think this should be added as a backlog task. @Claire-Bear

Comment (IscoRuta98) [reply web-client/src/app/views/delete-user/delete-user.page.ts:138]: @longtomjr and @Claire-Bear, I've added [this task](https://app.clickup.com/t/2f9a4mq) on the board and can be tackled after milestone 2

**Review by longtomjr — APPROVED** [review]

**Pull request merged**